doc/changelog/05-Ltac-language/20682-ltac1-depr-noninserted-implicits-Added.rst

```diff
@@ -0,0 +1,6 @@
+- **Added:**
+  flag :flag:`Ltac Always Insert Implicits` to interprete a :n:`@qualid` argument to a ltac as a term (with insertion of any implicit arguments, unfolding notations, etc)
+  in a toplevel :cmd:`Ltac` definition instead of interpreting it as the reference without implicit arguments.
```

Comment:
```suggestion
  flag :flag:`Ltac Always Insert Implicits` to interpret :n:`@qualid` arguments as terms (with insertion of any implicit arguments, unfolding notations, etc)
  in further toplevel :cmd:`Ltac` definitions instead of interpreting them as references without implicit arguments.
```